.github/workflows/global-ci-bundle.yml

```
@@ -593,80 +597,14 @@ jobs:
-          go-version: '1.25'
-
-      - name: Build Koncur
-        shell: bash
-        run: |
-          cd koncur
-          go build -o $HOME/.local/bin/koncur ./cmd/koncur/main.go
 
       - name: Run Koncur Kantra tests
-        shell: bash
-        working-directory: koncur
+        uses: ./koncur-kantra
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/koncur-kantra": line 4: unexpected key "type" for definition of input "ref"

(action)